src/stdatamodels/jwst/datamodels/tests/test_keyword_schemas.py

```diff
@@ -0,0 +1,137 @@
+"""
+Test duplicated schema information is in sync.
+"""
+
+import asdf
+import pytest
+
+# keyword schema file
+# - is this a single or multiple enum? (can it be inferred)
+# - what's the path (can it be inferred, no, because of the referencefile schema for name)
+# is there a pattern file (not all have them)
+# - all are a single regex
+# - what's the path? (can it be inferred)
+# core schema location
+# - single or multiple enum? (can it be inferred)
+# - what's the path
```

Comment:
This appears to be pseudocode for these tests.  Can this be removed, but the gist of it put into the docstring of the test itself, saying what it intends to do?

Reply:
I dropped these comments and added some basic docstrings in https://github.com/spacetelescope/stdatamodels/commit/88591ea16ca1ca01b1baa5c662bf8627a0bef334

How does that look?